comtypes/tools/codegenerator.py

```diff
@@ -276,12 +276,17 @@ def generate_code(self, items, filename):
         for line in wrapper.wrap(text):
             print(line, file=self.output)
 
-        if os.path.exists(filename):
```

Comment:
I thought it is part of the fix from @fusentasticus, isn't it?

Reply:
See PR #241

Reply:
What happened was when I did the initial PR for correcting the python3 problems is I used Python 3.8 and I installed comtypes using pypi.  This allowed the code conversion from python 2 to 3 to run. I then used the  generated  code and created a PR.

Me being brain dead at that time didn't think about any PR's that had been merged since 1.1.10 was released. The  PR I had submitted  got merged and that effectively undid 2 PR's that had gotten merged. Once of them you fixed and the other  that still needs to be fixed is this PR.

All I did was I looked at the code in the comtypes repo just prior to the Python3 fix getting merged and just after it getting merged and compared the differences. I looked for any changes that didn't make any sense for a Python 2 to 3 code conversion  and then looked to see if there was a PR that aligned with it. The result of that was this PR.

All this PR does is return the section of code that shouldn't have been changed back to what it was before the Python3 fix PR got merged, nothing more. If there are additional changes you are wanting to do above and beyond that I think should be done in a separate PR instead of this one. You are the boss and whatever you would like to do is what will get done.

I want to make sure that you understand fully what the purpose of this PR is before moving forward.

Reply:
Don't worry, I also rely on contributors testing because I'm usually overloaded. I'd prefer to close this PR for now. Let's see how new release will run on a wide audience. If this place will become problematic again, we can get back to it. Hope your own scenarios are not broken without this PR. This is what is good to know. Thanks for your efforts! Really appreciate it!